user-journeys/ansible-user-journey-maps.md

```diff
@@ -42,33 +42,78 @@ journey
 
 ### Beginner tasks
 
+- Sets up Ansible environment with vsCode plugins
 - Installs and/or updates ansible or ansible-core
```

Comment:
```suggestion
- Installs and/or updates ansible or ansible-core
- Installs Ansible Lint and developer tooling
```

Reply:
@samccann Just a thought. user updates are a package manager detail. content around updating and upgrading is more relevant to the operations user.